storage/lmdb/src/remap.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Support for memory map reallocation
+
+use logging::log;
+use utils::tap_error_log::LogError;
+
+/// Represents LMDB memory map size
+#[derive(Eq, PartialEq, PartialOrd, Ord, Clone, Copy, Debug)]
+pub struct MemSize(usize);
```

Comment:
Using usize here isn't really a good idea... if for any reason it's 32-bit (like if we ever decide to support 32-bit), everything will fall apart. Please use u128 or u64 at least.

I know lmdb-rs uses usize... but this is a mistake. I already got into trouble because of this in C++.